src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -349,14 +349,67 @@ export default function ({
     });
   }
 
+  function hasOneOf(value: unknown): value is { __one_of: ConditionalTemplateType[] } {
+    return typeof value === 'object' && value !== null && '__one_of' in value;
```

Comment:
`value && '_ one_of' in value` is enough

Reply:
ts complains if I leave out the `typeof` check. ``` Object is of type 'unknown'.```. Perhaps I am missing something?